### ISSUE TITLE
Diluent parameter understanding

### ISSUE THREAD

**Issue by rde-girl** [hitranonline/hapi#33]
Hello, 

I am a new user of HAPI, and I am having trouble understanding what the diluent parameter is.
I have read the HAPI manual and tried to research articles online, and I understand what this does to the line function broadening in relation to gamma. However, I still don't understand what this actually means in relation to species I am considering

For example, if I am working with H2O (all isotopes), how much H2O is there in relation to the broadening mixture?
In other words, what does it means to set diluent = {'self' : 1} or diluent = {'air' : 1} or diluent = {'H2O': 1} in relation to the absorbing species? Note that I get different results for each of these. If I just have a reference cell filled with H2O, do I need a broadening mixture?

**Comment by erwanp** [comment]
If you have a reference cell filled with H2O, your H2O mole_fraction should be 1 and the diluent mole_fraction will be 0, it should have no influence. You're saying you get different results : what's your H2O mole fraction ?
